src/main/java/com/github/hi_fi/httpclient/cookie/Cookie.java

```diff
@@ -0,0 +1,31 @@
+package com.github.hi_fi.httpclient.cookie;
+
+import java.util.LinkedHashMap;
+
+public class Cookie extends LinkedHashMap<String, String> {
+
+    private static final long serialVersionUID = -5706229460116409983L;
+
+    public Cookie(String setCookieValue) {
+        parse(setCookieValue);
+    }
+
+    public String name() {
+        return size() == 0 ? "" : entrySet().iterator().next().getKey();
+    }
+
+    public String value() {
+        return size() == 0 ? "" : entrySet().iterator().next().getValue();

```

Comment:
By the way, if `value()` is identical to `name()` why not just invoke one from another?

Reply:
> By the way, if `value()` is identical to `name()` why not just invoke one from another?

Hi,
I am not quite sure that I understand you. 
Actually, name() and value() are different: getKey() vs getValue() respectively.
These values are derived from Set-Cookie header's value by splitting it into 2 parts by first '=' symbol.

Reply:
> Actually, name() and value() are different: getKey() vs getValue() respectively.

I see it now, thanks! Then disregard my comment.